src/Network/Network.Test/ScenarioTests/NetworkManagerTests.cs

```diff
@@ -98,5 +98,53 @@ public void NetworkManagerResourceMinimumParameterCreate()
         {
             TestRunner.RunTestScript("Test-NetworkManagerResourceMinimumParameterCreate");
         }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.Owner, NrpTeamAlias.nsmdev)]
+        public void NetworkManagerIpamPoolCRUD()
+        {
+            TestRunner.RunTestScript("Test-NetworkManagerIpamPoolCRUD");
+        }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.Owner, NrpTeamAlias.nsmdev)]
+        public void NetworkManagerIpamPoolStaticCidrCRUD()
+        {
+            TestRunner.RunTestScript("Test-NetworkManagerIpamPoolStaticCidrCRUD");
+        }
+
+        [Fact]
```

Comment:
you can combine all of these

Reply:
As in combine the references to the tests i.e:
        public void NetworkManagerIpamPoolAndStaticCidrCRUD()
        {
            TestRunner.RunTestScript("Test-NetworkManagerIpamPoolCRUD");
            TestRunner.RunTestScript("Test-NetworkManagerIpamPoolStaticCidrCRUD");
        }
Or actually combine them all in the ps1 file?

Reply:
I was referring to the vnet verifier tests; it would make sense to do all crud in one, since to create a run you need intent+workspace anyways